### PR TITLE
Add sequence name to metadata extracted for heuristics

### PR DESCRIPTION
**Pull request by leej3**
Although its stored at different locations in the DICOM depending on scanner manufacturer I have found the sequence name to be useful for developing heuristics. Might be worth including...

### CODE REVIEW

File: bin/heudiconv [hunk 1]
```diff
@@ -272,7 +273,15 @@ def process_dicoms(fl, dcmsession=None, basedir=None, dcmfilter=None):
             refphys = dcminfo.ReferringPhysicianName
         except AttributeError:
             refphys = '-'
-
+        if dcminfo.get([0x18,0x24], None):
+            # GE and Philips scanners
```
Comment (yarikoptic) [inline bin/heudiconv]: Styling: Regular variables shouldn't be capitalized as classes. Make it eg `sequence_name `

**Comment by satra** [comment]
@leej3 - we are soon going to merge #32 . perhaps you can redo this request after rebasing/merging off master after that merge takes place.

**Comment by leej3** [comment]
yep, will do

**Comment by satra** [comment]
@leej3 - could you please update this, now that yarik's PR has been merged.

**Pull request merged**